app/models/mhv_account.rb

```diff
@@ -69,7 +82,11 @@ def terms_and_conditions_accepted?
   end
 
   def preexisting_account?
-    user&.mhv_correlation_id.present?
+    user&.mhv_correlation_id.present? && !previously_registered?
```

Comment:
what about !previously_upgraded? or creation/upgrade failed steps?

previously_registered? only returns true if vets.gov created the account without upgrading it right?

should we also have some special handing for the 'failed create' 'failed upgrade' steps?

Reply:
And if the account was existing before (didn't register on Vets.gov) and failed to upgrade when we were upgrading everyone, they'll now be considered "existing" and won't get upgraded.